reviewers/att_no_white_spaces.go

```diff
@@ -0,0 +1,42 @@
+package reviewers
+
+import (
+	"bufio"
+	"io"
+	"regexp"
+	"strings"
+)
+
+type AttrNoWhiteSpaces struct{}
+
+func (at AttrNoWhiteSpaces) ReviewerName() string {
+	return "attribute/no-white-spaces"
+}
+
+func (at AttrNoWhiteSpaces) Review(path string, page io.Reader) ([]Fault, error) {
+	result := []Fault{}
+
+	var number int
+	var line string
+
+	scanner := bufio.NewScanner(page)
+	for scanner.Scan() {
+		number++
+
+		line = scanner.Text()
+		if strings.TrimSpace(line) == "" {
+			continue
+		}
+
+		exp := regexp.MustCompile(`[a-z\-]+(\s+\=\s*|\s*\=\s+)`)
```

Comment:
@tatang26 is this regexp covering cases for attributes in uppercase?

Reply:
@larrymjordan no, because this case is covered by [0013]

Reply:
I can add this case if you think that is needed

Reply:
@tatang26 I thought the same. You could just strings.ToLower the line.

Reply:
And Yes. It is needed :)

Reply:
@paganotoni @larrymjordan Done!

Reply:
This regular expression will not work for H1 :)

Reply:
@tatang26 I meant that if the attribute has a number it will not work. something like `data-attribute2 ="[UUID]"` may not match that expression.

Reply:
@paganotoni the expresion is updated to validate attributes with numbers

Reply:
Ah nice. I was kind of wanting that you notice the following. This rule is in charge of detecting the space between the = sign and whatever the name of value of the attribute are. IMO for this reviewer could be anything in the name of the key, in that sense I would suggest that the expression do not limit the names selecting the character, rather it should accept anything that is not a space.

Some cases I think we should add in the tests:
- `<img data-name ="sss">`
- `<img data-4name ="sss">`
- `<img data**4name ="sss">`

Reply:
@tatang26 sorry for too much back and forth with this one

Reply:
@paganotoni Don't worry I was thinking about create another PR with those changes 😅